templates/verify.html

```diff
@@ -31,6 +31,14 @@
     </style>
   </head>
   <body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0" style="background-color:#F2F2F2;width:100% !important;height:100% !important;padding:0;margin:0;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%;">
+    <div itemscope itemtype="http://schema.org/EmailMessage">
+      <div itemprop="potentialAction" itemscope itemtype="http://schema.org/ViewAction">
+        <link itemprop="target" href="{{{link}}}"/>
+        <meta itemprop="name" content="{{t 'Complete sign-up'}}"/>
+        <meta itemprop="url" content="{{{link}}}"/>
```

Comment:
If no one has any other ideas, let's just add `one_click=true` to that url. I guess we can track it in the server logs for now. just make sure the generated url with `one_click` is a valid url.